nuxeo-distribution/nuxeo-wizard-tests/src/test/java/org/nuxeo/ftest/wizard/ITWizardAndUpdateCenterTests.java

```diff
@@ -152,7 +149,7 @@ public void runWizardAndRestart() throws Exception {
         userPage = userPage.navById(WizardPage.class, "checkNetwork");
         assertTrue(userPage.hasError());
         userPage.clearInput("nuxeo.ldap.url");
-        userPage.fillInput("nuxeo.ldap.url", "ldap://ldap.testathon.net:389");
+        userPage.fillInput("nuxeo.ldap.url", "ldap://ldap.forumsys.com:389");
```

Comment:
Can't we find a host under our control to test LDAP connectivity rather than something on the internet?